pkg/controller/account/iam.go

```diff
@@ -493,6 +493,7 @@ func CreateIAMUser(reqLogger logr.Logger, client awsclient.Client, userName stri
 				return &iam.CreateUserOutput{}, err
 			}
 		}
+		break
```

Comment:
If we break here without adding an explicit continue in the first aerr case above this will break out of the for loop every time. So we can either add a continue there or we need to wrap this break in an “else” block

Reply:
good catch. again.